tests/factories/item_factories.py

```diff
@@ -14,10 +14,4 @@ class ItemFactory(SQLAlchemyModelFactory):
     author = 'Scott Gladstone'
     summary = 'Summary of the feed item.'
     text = 'Text of the feed item.'
-    feed = factory.SubFactory(FeedFactory)
-
-class EbolaItemFactory(ItemFactory):
```

Comment:
If you insist. It just makes test_categorization.py so messy, so i thought it might be nice to extract it since it's such a hefty string.